### PR TITLE
Script for introduction-to-ethical-hacking.md

### PR DESCRIPTION
**Pull request by mrpatil007**
This is Script for the first video of Introduction of Ethical Hacking

### CODE REVIEW

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+Hello Hacker, I am Bhushan Patil and this is first introduction + some tips video. so why are you waiting for let's Start 
+to learn hacking.
+
+### What is Ethical Hacking ?
+    So, Ethical Hacking means to penetrate a computer system or network with some Ethicks
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Ethicks~~ Ethics

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Hello Hacker, I am Bhushan Patil and this is first introduction + some tips video. so why are you waiting for let's Start 
+to learn hacking.
+
+### What is Ethical Hacking ?
+    So, Ethical Hacking means to penetrate a computer system or network with some Ethicks 
+    those type of hacker called as Ethical Hacker. so what are thus ethics you need to follow
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- those type of hacker called as Ethical Hacker. so what are thus ethics you need to follow 
+ 
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+Hello Hacker, I am Bhushan Patil and this is first introduction + some tips video. so why are you waiting for let's Start 
+to learn hacking.
+
+### What is Ethical Hacking ?
+    So, Ethical Hacking means to penetrate a computer system or network with some Ethicks 
+    those type of hacker called as Ethical Hacker. so what are thus ethics you need to follow 
+    for being a ethical hacker is mention as follows:
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- for being a ethical hacker is mention as follows:
- You need to follow these Ethics:
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        1. You respect the individual's or company's privacy
+        2. You close out your work, not leaving anything open for you 
+           or someone else to exploit at a later time
+        3. Just give acknowledge to the Developer or Hardware Manufacture About there Security Bug
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~About there Security Bug~~ about their Security Bug

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    So, Ethical Hacking means to penetrate a computer system or network with some Ethicks 
+    those type of hacker called as Ethical Hacker. so what are thus ethics you need to follow 
+    for being a ethical hacker is mention as follows:
+        1. You respect the individual's or company's privacy
+        2. You close out your work, not leaving anything open for you 
+           or someone else to exploit at a later time
+        3. Just give acknowledge to the Developer or Hardware Manufacture About there Security Bug 
+        
+### What are the opportunities as a Ethical hacker ?
+    Hm, There are lot's of opportunities as a Ethical Hacker because in Digital World
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- Hm, There are lot's of opportunities as a Ethical Hacker because in Digital World
+ There are lots of opportunities as a Ethical Hacker because in the Digital World
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    those type of hacker called as Ethical Hacker. so what are thus ethics you need to follow 
+    for being a ethical hacker is mention as follows:
+        1. You respect the individual's or company's privacy
+        2. You close out your work, not leaving anything open for you 
+           or someone else to exploit at a later time
+        3. Just give acknowledge to the Developer or Hardware Manufacture About there Security Bug 
+        
+### What are the opportunities as a Ethical hacker ?
+    Hm, There are lot's of opportunities as a Ethical Hacker because in Digital World 
+    Managing and Securing Data is the main objective so as a Ethical Hacker you have
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- Managing and Securing Data is the main objective so as a Ethical Hacker you have
+ Managing and Securing Data is one of the main objectives
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+    for being a ethical hacker is mention as follows:
+        1. You respect the individual's or company's privacy
+        2. You close out your work, not leaving anything open for you 
+           or someone else to exploit at a later time
+        3. Just give acknowledge to the Developer or Hardware Manufacture About there Security Bug 
+        
+### What are the opportunities as a Ethical hacker ?
+    Hm, There are lot's of opportunities as a Ethical Hacker because in Digital World 
+    Managing and Securing Data is the main objective so as a Ethical Hacker you have 
+    a big Opportunities in ITsec Field
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- a big Opportunities in ITsec Field 
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+    those categories of hackers based on their behaviors.
+    
+    1.White Hat Hacker : These type of Hacker called as "Ethical Hacker". Those type of hackers 
+    experts in compromising computer security systems who use their abilities for good, ethical, 
+    and legal purposes.
+    
+    2.Black Hat Hacker : These type of hacker mainly popular on Media. they violate computer security 
+    for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+    identity thieves)
+
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: Switch Grey and Black around :wink:

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+    identity thieves)
+    
+    3.Grey Hat Hacker  : These type of hacker test the system without the permission of Owner and after 
+    testing they inform the Owner about the system flaw for there personal gain 
+    
+### Important Weapon (Tools) to be need for Hacking
+    Kali Linux : Operating System design for Hacker's and Pentester it has a bunch of tools to play :)
+    Link to Download :https://www.kali.org/downloads/
+    
+    Note : You Can also Go with other linux flavours develop for Hackings Like Bugtraq, arch and etc
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Note : You Can also Go with other linux flavours develop for Hackings Like Bugtraq, arch and etc~~
There also other linux flavours such as Bugtraq, Arch etc.

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+    2.Black Hat Hacker : These type of hacker mainly popular on Media. they violate computer security 
+    for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+    identity thieves)
+    
+    3.Grey Hat Hacker  : These type of hacker test the system without the permission of Owner and after 
+    testing they inform the Owner about the system flaw for there personal gain 
+    
+### Important Weapon (Tools) to be need for Hacking
+    Kali Linux : Operating System design for Hacker's and Pentester it has a bunch of tools to play :)
+    Link to Download :https://www.kali.org/downloads/
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- Link to Download :https://www.kali.org/downloads/
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+    
+    2.Black Hat Hacker : These type of hacker mainly popular on Media. they violate computer security 
+    for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+    identity thieves)
+    
+    3.Grey Hat Hacker  : These type of hacker test the system without the permission of Owner and after 
+    testing they inform the Owner about the system flaw for there personal gain 
+    
+### Important Weapon (Tools) to be need for Hacking
+    Kali Linux : Operating System design for Hacker's and Pentester it has a bunch of tools to play :)
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Kali Linux~~ `[Kali Linux](https://www.kali.org)`

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+
+### Types or Categories of hacker's
+    Basically there are Three categories of hacker's that is => "White Hat", "Black Hat" and "Grey Hat"
+    those categories of hackers based on their behaviors.
+    
+    1.White Hat Hacker : These type of Hacker called as "Ethical Hacker". Those type of hackers 
+    experts in compromising computer security systems who use their abilities for good, ethical, 
+    and legal purposes.
+    
+    2.Black Hat Hacker : These type of hacker mainly popular on Media. they violate computer security
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~These type of hacker~~ These type of hackers
~~they violate computer~~ They violate computer

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+    
+    1.White Hat Hacker : These type of Hacker called as "Ethical Hacker". Those type of hackers 
+    experts in compromising computer security systems who use their abilities for good, ethical, 
+    and legal purposes.
+    
+    2.Black Hat Hacker : These type of hacker mainly popular on Media. they violate computer security 
+    for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+    identity thieves)
+    
+    3.Grey Hat Hacker  : These type of hacker test the system without the permission of Owner and after
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~3.Grey Hat Hacker  :~~ 3. Gray Hat Hacker:

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+### What are the opportunities as a Ethical hacker ?
+    Hm, There are lot's of opportunities as a Ethical Hacker because in Digital World 
+    Managing and Securing Data is the main objective so as a Ethical Hacker you have 
+    a big Opportunities in ITsec Field 
+
+### Types or Categories of hacker's
+    Basically there are Three categories of hacker's that is => "White Hat", "Black Hat" and "Grey Hat"
+    those categories of hackers based on their behaviors.
+    
+    1.White Hat Hacker : These type of Hacker called as "Ethical Hacker". Those type of hackers
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~1.White Hat Hacker : These type of Hacker called as "Ethical Hacker".~~ 1. White Hat Hacker: These type of hackers are often known as "Ethical Hackers".

Comment (mrpatil007) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: @atjonathan Thanks for correcting typo mistake

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 15]
```diff
@@ -0,0 +1,38 @@
+Hello Hacker, I am Bhushan Patil and this is first introduction + some tips video. so why are you waiting for let's Start 
+to learn hacking.
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- to learn hacking.
```

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 16]
```diff
@@ -0,0 +1,38 @@
+Hello Hacker, I am Bhushan Patil and this is first introduction + some tips video. so why are you waiting for let's Start
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Hello Hacker, I am Bhushan Patil and this is first introduction + some tips video. so why are you waiting for let's Start~~ Hey Camper, I am Bhushan Patil and this is an introduction and some tips about this series on Ethical Hacking.

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md (hunk 15 above)
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]:
```diff
- to learn hacking.
```

**Comment by atjonathan** [comment]
@mrpatil007, also please remove the tabs. Sorry for the large quantity of comments :disappointed:

**Comment by mrpatil007** [comment]
@atjonathan ok i will make the changes it will teach me a lot to for overcome the future mistakes thanks man

**Comment by atjonathan** [comment]
@mrpatil007 :smiley: No Problem, thanks a lot for your work on this series so far :tada:

**Comment by mrpatil007** [comment]
@atjonathan which tabs i want to remove ?

**Comment by atjonathan** [comment]
@mrpatil007 all of the ones before a sentence after each heading.

**Comment by mrpatil007** [comment]
@atjonathan i fix the changes

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
+Hey Camper, I am Bhushan Patil and this is an introduction and some tips about this series on Ethical Hacking. So why are you waiting for let's Start
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~So why are you waiting for let's Start~~

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+2.Black Hat Hacker: These type of hackers mainly popular on Media. They violate computer security 
+for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+identity thieves)
+    
+3.Grey Hat Hacker: These type of hacker test the system without the permission of Owner and after 
+testing they inform the Owner about the system flaw for there personal gain 
+    
+### Important Weapon (Tools) to be need for Hacking
+Kali Linux: Operating System design for Hacker's and Pentester it has a bunch of tools to play :)
+Link to Download :[Kali Linux](https://www.kali.org)
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Link to Download :~~

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 19]
```diff
@@ -0,0 +1,36 @@
...
+    
+2.Black Hat Hacker: These type of hackers mainly popular on Media. They violate computer security 
+for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+identity thieves)
+    
+3.Grey Hat Hacker: These type of hacker test the system without the permission of Owner and after 
+testing they inform the Owner about the system flaw for there personal gain 
+    
+### Important Weapon (Tools) to be need for Hacking
+Kali Linux: Operating System design for Hacker's and Pentester it has a bunch of tools to play :)
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Kali Linux:~~ [Kali Linux](https://www.kali.org) :

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+    1. You respect the individual's or company's privacy
+    2. You close out your work, not leaving anything open for you 
+       or someone else to exploit at a later time
+    3. Just give acknowledge to the Developer or Hardware Manufacture about their Security Bug 
+    
+### What are the opportunities as a Ethical hacker ?
+There are lots of opportunities as a Ethical Hacker because in the Digital World
+Managing and Securing Data is one of the main objectives so as a Ethical Hacker
+
+### Types or Categories of hacker's
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~hacker's~~ Hackers

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+
+### What is Ethical Hacking ?
+So, Ethical Hacking means to penetrate a computer system or network with some Ethics 
+You need to follow these:
+    1. You respect the individual's or company's privacy
+    2. You close out your work, not leaving anything open for you 
+       or someone else to exploit at a later time
+    3. Just give acknowledge to the Developer or Hardware Manufacture about their Security Bug 
+    
+### What are the opportunities as a Ethical hacker ?
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~hacker ?~~ Hacker?

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 22]
```diff
@@ -0,0 +1,36 @@
+Hey Camper, I am Bhushan Patil and this is an introduction and some tips about this series on Ethical Hacking. So why are you waiting for let's Start 
+
+### What is Ethical Hacking ?
+So, Ethical Hacking means to penetrate a computer system or network with some Ethics
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~Ethics~~ Ethics.

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 23]
```diff
@@ -0,0 +1,36 @@
...
+    2. You close out your work, not leaving anything open for you 
+       or someone else to exploit at a later time
+    3. Just give acknowledge to the Developer or Hardware Manufacture about their Security Bug 
+    
+### What are the opportunities as a Ethical hacker ?
+There are lots of opportunities as a Ethical Hacker because in the Digital World
+Managing and Securing Data is one of the main objectives so as a Ethical Hacker
+
+### Types or Categories of hacker's
+Basically there are Three categories of hacker's that is => "White Hat", "Black Hat" and "Grey Hat"
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~"Grey Hat"~~ "Gray Hat"

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 24]
```diff
@@ -0,0 +1,36 @@
...
+    
+1.White Hat Hacker:White Hat Hacker: These type of hackers are often known as "Ethical Hackers". Those type of hackers 
+experts in compromising computer security systems who use their abilities for good, ethical, 
+and legal purposes.
+    
+2.Black Hat Hacker: These type of hackers mainly popular on Media. They violate computer security 
+for personal gain (such as stealing credit card numbers or harvesting personal data for sale to 
+identity thieves)
+    
+3.Grey Hat Hacker: These type of hacker test the system without the permission of Owner and after
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~3.Grey Hat Hacker:~~ 3.Gray Hat Hacker:

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 25]
```diff
@@ -0,0 +1,36 @@
...
+
+### Types or Categories of hacker's
+Basically there are Three categories of hacker's that is => "White Hat", "Black Hat" and "Grey Hat"
+those categories of hackers based on their behaviors.
+    
+1.White Hat Hacker:White Hat Hacker: These type of hackers are often known as "Ethical Hackers". Those type of hackers 
+experts in compromising computer security systems who use their abilities for good, ethical, 
+and legal purposes.
+    
+2.Black Hat Hacker: These type of hackers mainly popular on Media. They violate computer security
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~2.Black Hat Hacker:~~ 2. Black Hat Hacker:

File: video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md [hunk 26]
```diff
@@ -0,0 +1,36 @@
...
+    
+### What are the opportunities as a Ethical hacker ?
+There are lots of opportunities as a Ethical Hacker because in the Digital World
+Managing and Securing Data is one of the main objectives so as a Ethical Hacker
+
+### Types or Categories of hacker's
+Basically there are Three categories of hacker's that is => "White Hat", "Black Hat" and "Grey Hat"
+those categories of hackers based on their behaviors.
+    
+1.White Hat Hacker:White Hat Hacker: These type of hackers are often known as "Ethical Hackers". Those type of hackers
```
Comment (atjonathan) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: ~~1.White Hat Hacker:White Hat Hacker:~~ 1. White Hat Hacker:

Comment (mrpatil007) [inline video-categories/ethical-hacking/scripts/introduction-to-ethical-hacking.md]: @atjonathan Thanks

**Comment by mrpatil007** [comment]
Finally done learn a lot

**Comment by mrpatil007** [comment]
@atjonathan  please overlook the script and merge it

**Comment by atjonathan** [comment]
#### LGTM :+1: :tada:

**Pull request merged**